frontend/src/__tests__/e2e/CreateEditDeleteProject.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { test, expect, Page } from '@playwright/test';
+
+const cleanTestProject = async (page: Page) => {
+  await page.goto('/projects');
+  const count = await page.getByRole('link', { name: process.env.TEST_PROJECT_NAME }).count();
+  if (count > 0) {
+    await page
+      .locator('tr', {
+        has: page.locator(`text="${process.env.TEST_PROJECT_NAME}"`),
```

Comment:
Let us not refer to the process var directly -- please put it in the constants files

Reply:
It seems we have a horrible organization on this front too 🤦 More tech debt. 

Today the constants are in here: `frontend/src/utilities/const.ts` -- it's awkward, but please put it in there too.

Reply:
Issues with playwright and typescript path mapping